content/docs/started/getting-started-icp.md

```diff
@@ -69,51 +69,51 @@ NAME         STATUS    ROLES                      AGE       VERSION
    
     Some Kubeflow components need PVs to storage data, such as minio, mysql katib. We need to create PVs for those pods in advance. 
     The PVs defination file (`pv.yaml`) is as following:
-
-     ```yaml
-      apiVersion: v1
-      kind: PersistentVolume
-      metadata:
-        name: kubeflow-pv1
-      spec:
-        capacity:
-          storage: 20Gi
-        accessModes:
-        - ReadWriteOnce
-        persistentVolumeReclaimPolicy: Retain
-        nfs:
-          path: ${NFS_SHARED_DIR}/pv1
-          server: ${NFS_SERVER_IP}
-      ---
-      apiVersion: v1
-      kind: PersistentVolume
-      metadata:
-        name: kubeflow-pv2
-      spec:
-        capacity:
-          storage: 20Gi
-        accessModes:
-        - ReadWriteOnce
-        persistentVolumeReclaimPolicy: Retain
-        nfs:
-          path: ${NFS_SHARED_DIR}/pv2
-          server: ${NFS_SERVER_IP}
-      ---
-      apiVersion: v1
-      kind: PersistentVolume
-      metadata:
-        name: kubeflow-pv3
-      spec:
-        capacity:
-          storage: 20Gi
-        accessModes:
-        - ReadWriteOnce
-        persistentVolumeReclaimPolicy: Retain
-        nfs:
-          path: ${NFS_SHARED_DIR}/pv3
-          server: ${NFS_SERVER_IP}
-     ```
-
+    
+    ```yaml
```

Comment:
Suggestion: Copy the indentation of the image policy file on line 45.

Reply:
It works in website now after I deleted all blanks before each line. :-)